avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1872,3 +1816,155 @@ def analysisPlots(particlesList, fieldsList, cfg, demOri, dem, outDir):
     fig3, ax3 = plotPosition(
         fig3, ax3, partEnd, demOri, fieldEnd['P']/1000, pU.cmapPres, 'kPa', plotPart=False)
     plt.show()
+
+
+def prepareVarSimDict(standardCfg, inputSimFiles, variationDict, varyAll=True):
+    """ Prepare a dictionary with simulations that shall be run with varying parameters following the variation dict
+
+        Parameters
+        -----------
+        standardCfg : configParser object
+            default configuration or local configuration
+        inputSimFiles: dict
+            info dict on available input data
+        variationDict: dict
+            dictionary with parameter to be varied as key and list of it's values
+        varyAll: boolean
+            True if all possible parameter combinations shall be performed
+            False if only one parameter shall be varied and the others from standard configuration
+
+        Returns
+        -------
+        simDict: dict
+            dicionary with info on simHash, releaseScenario, release area file path, simType and contains full configuration
+            configparser object for simulation run
+
+    """
+
+    # get list of simulation types
+    if 'simTypeList' in variationDict:
+        if isinstance(variationDict['simTypeList'], str):
+            simTypeList = [variationDict['simTypeList']]
+        else:
+            simTypeList = variationDict['simTypeList']
+        del variationDict['simTypeList']
```

Comment:
if the simTypeList in the variationDict is only a string so only one simType is provided, it is converted to a list

Reply:
I would suggest that we make sure that variationDict['simTypeList'] alread is a list, even if it only has one element. Especially since the name strongly suggests it is a list...

Reply:
if it is created through the deriveParamterSet.py it is for sure a list, but if it is provided from outside to runCom1DFA.py not for sure, shall we make this a requirement?